Assignment/Program.cs

```diff
@@ -1,7 +1,50 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿// Check strong password
+var number = new List<int> { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 };
+var capitalLetter = new List<char> { 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z' };
+var smallLetter = new List<char> { 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z' };
+var symbol = new List<char> { ':', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '-', '+' };
+int count = 0;
+Console.WriteLine ("Enter your password : ");
+var passwordInput = Console.ReadLine ();
+var password = passwordInput!.ToCharArray ();
+if (password.Length >= 6) {
+   count = count + 1;
+   int capitalCount = 0;
+   for (int i = 0; i < password.Length; i++) {

```

Comment:
Unnecessarily overcomplicated code with many for loops.

Can you think of doing with 1 for loop?